### ISSUE TITLE
ads between and during videos

### ISSUE THREAD

**Issue by marcelodeepspacepodcast** [hopollo/OBS-Youtube-Player#20]
hello guys!
since yesterday it started to playing ads between and during the videos, so now we need to subscribe to youtube premium or is something in the hopollo?

**Comment by DamianQualshy** [comment]
that's because youtube now puts ads in embeds, nothing can be done about it afaik

**Comment by hopollo** [comment]
Yeah i saw that recently that's a big issue even game breaking for the tool tho

**Comment by jasperfrontend** [comment]
I know this is a graveyard reply and I'm sorry for that. If you have YouTube premium, change your OBS browser source URL to youtube.com, then right-click it > interact and log in on YouTube with your premium account. Then change the browser source URL back to the hopollo player and it won't show ads in embeds anymore. Since I make some money on my streams, the Premium basically pays itself back.
